### ISSUE TITLE
How to use other size image?

### ISSUE THREAD

**Issue by qqwrz114** [scaelles/OSVOS-TensorFlow#11]
Is this model used for specific image size(480P)? 
I tried 960x640 to train in osvos_demo.py, it failed.
when i change the image size to 640x480,it work.

**Comment by gdshen** [comment]
I think you should try a smaller learning rate, or the gradient will explode.

**Comment by kmaninis** [comment]
Yes, the loss is not normalized to the image resolution. I would suggest either using a smaller learning rate or dividing the loss by the size of the output and using a universal (way smaller) loss.
